### PR TITLE
SGD8-446: Add translations for from...to...year

### PR DESCRIPTION
**Pull request by hstandaert**
## Description
Add translation for range slider. 

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have updated the documentation accordingly.
- [ ] I have updated the style guide CHANGELOG accordingly.
- [ ] I have updated the gent_base CHANGELOG accordingly.
- [ ] I have ran gulp axe on the compiled files and found no errors.

### CODE REVIEW

**Review by Jeroen005 — APPROVED** [review]

File: translations/nl.po [hunk 1]
```diff
@@ -393,13 +401,12 @@ msgstr "Gepost op"
...
 
-#: themes/contrib/gent_base/templates/custom/nodes/node--terrain--default.html.twig:4;25
+#: themes/contrib/gent_base/templates/custom/nodes/node--terrain--default.html.twig:4;24
 msgid "Park offer"
 msgstr "Aanbod en mogelijkheden"
 
-#: themes/contrib/gent_base/templates/custom/nodes/node--terrain--default.html.twig:14
-msgid "Description"
-msgstr "Beschrijving"
+#~ msgid "Description"
```
Comment (MPParsley) [inline translations/nl.po]: comments mag je verwijderen

File: templates/contrib/facets/facets-item-list--range-slider.html.twig [hunk 2]
```diff
@@ -38,11 +38,8 @@
     {{ items.value }}
 
     <div class="pips-preview">
-      {{ 'From'|t }}
-      <span class="pips-preview-min"></span>
-      {{ 'to'|t }}
-      <span class="pips-preview-max"></span>
-      {{ 'year'|t }}
+
+      {{ 'From <span class="pips-preview-min"></span> to <span class="pips-preview-max"></span> year'|t }}
```
Comment (MPParsley) [inline templates/contrib/facets/facets-item-list--range-slider.html.twig:42]: Bestaat er geen nettere manier om dit te doen? Zoals dit of is er geen ondersteuning voor placeholders in twig voor Drupal:

    {% set min = "<span class='pips-preview-min'></span>" %}
    {% set max = "<span class='pips-preview-max'></span>" %}

    {% trans %}
      From {{ min|raw }} to {{ min|raw }} year
    {% endtrans %}

File: translations/general.pot [hunk 3]
```diff
@@ -41,7 +42,7 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: PROJECT VERSION\n"
-"POT-Creation-Date: 2018-10-15 12:07+0000\n"
+"POT-Creation-Date: 2018-10-25 13:07+0200\n"
 "PO-Revision-Date: YYYY-mm-DD HH:MM+ZZZZ\n"
 "Last-Translator: NAME <EMAIL@ADDRESS>\n"
```
Comment (MPParsley) [inline translations/general.pot:47]: Best in POEdit eens je naam en email instellen bij instellingen:

poedit

**Review by MPParsley — APPROVED** [review]

File: templates/contrib/facets/facets-item-list--range-slider.html.twig (hunk 2 above)
Comment (hstandaert) [reply templates/contrib/facets/facets-item-list--range-slider.html.twig:42]: @MPParsley, laten we dit dan voorlopig hoe het is? ik vind niet meteen een betere werkbare oplossing.

Comment (MPParsley) [reply templates/contrib/facets/facets-item-list--range-slider.html.twig:42]: Tuurlijk, geen probleem.

File: translations/nl.po [hunk 4]
```diff
@@ -393,14 +401,10 @@ msgstr "Gepost op"
...
 msgid "Park offer"
 msgstr "Aanbod en mogelijkheden"
 
-#: themes/contrib/gent_base/templates/custom/nodes/node--terrain--default.html.twig:14
-msgid "Description"
-msgstr "Beschrijving"
-
 #~ msgid "1 result found"
 #~ msgid_plural "@count results found"
 #~ msgstr[0] "1 resultaat gevonden"
```
Comment (MPParsley) [inline translations/nl.po]:
```suggestion
```

File: translations/nl.po [hunk 5]
```diff
@@ -393,14 +401,10 @@ msgstr "Gepost op"
...
+#: themes/contrib/gent_base/templates/custom/nodes/node--terrain--default.html.twig:4;24
 msgid "Park offer"
 msgstr "Aanbod en mogelijkheden"
 
-#: themes/contrib/gent_base/templates/custom/nodes/node--terrain--default.html.twig:14
-msgid "Description"
-msgstr "Beschrijving"
-
 #~ msgid "1 result found"
 #~ msgid_plural "@count results found"
```
Comment (MPParsley) [inline translations/nl.po]:
```suggestion
```

File: translations/nl.po [hunk 6]
```diff
@@ -393,14 +401,10 @@ msgstr "Gepost op"
...
-#: themes/contrib/gent_base/templates/custom/nodes/node--terrain--default.html.twig:4;25
+#: themes/contrib/gent_base/templates/custom/nodes/node--terrain--default.html.twig:4;24
 msgid "Park offer"
 msgstr "Aanbod en mogelijkheden"
 
-#: themes/contrib/gent_base/templates/custom/nodes/node--terrain--default.html.twig:14
-msgid "Description"
-msgstr "Beschrijving"
-
 #~ msgid "1 result found"
```
Comment (MPParsley) [inline translations/nl.po]:
```suggestion
```

**Pull request merged**